src/constants/Constants.js

```diff
@@ -190,4 +195,10 @@ export default class Constants {
       label: "English",
     },
   };
+
+  static VALIDATION_TYPE = {
+    PATTERN: "pattern",
```

Comment:
Ohh... I see, but why do you need "pattern"

I understand it in the following way:
- to solve pattern is **required in order to save** and will never be useful to complete
- some questions need to be filled in, i.e. **required in order to save** 
- some questions need to be filled in, i.e. **required in order to save** 

We could differentiate this validation type as being not PATTERN but INVALID, but I do not see an advantage of that right now.

Reply:
Indeed it is not really needed for now, I was just thinking about possible future validators that would need this without relying on the question properties, but it is probably overkill for now

Reply:
No, the point is to differentiate the validation types, as it is needed to get the correct color. If it makes more sense, we could rename it to `validationColor: red | orange`

Reply:
@blcham so is it question property? The way it is done now, it is only attached to the validator itself

Reply:
So I mean no need to have URI but simple object with string values as it is now